File: mobile/src/main/java/tr/xip/prayertimes/api/objects/Country.java
package tr.xip.prayertimes.api.objects;

import java.io.Serializable;

public class Country implements Serializable {
    int databaseId;
    private String id;
    private String name;

    public Country(String id, String name) {
        this.id = id;
        this.name = name;
    }

    public Country(int databaseId, String id, String name) {
        this.databaseId = databaseId;
        this.id = id;
        this.name = name;
    }

    public int getDatabaseId() {
        return databaseId;
    }

    public String getId() {
        return id;
    }

    public String getName() {
        return name;
    }
}

File: mobile/src/main/kotlin/tr/xip/prayertimes/model/Country.kt
package tr.xip.prayertimes.model

import java.io.Serializable

class Country : Serializable {
    var databaseId: Int = 0
        internal set
    var id: String? = null
    var name: String? = null

    constructor(id: String, name: String) {
        this.id = id
        this.name = name
    }

    constructor(databaseId: Int, id: String, name: String) {
        this.databaseId = databaseId
        this.id = id
        this.name = name
    }
}
